apps/rush-lib/src/logic/taskRunner/TaskCollection.ts

```diff
@@ -105,9 +106,7 @@ export class TaskCollection {
     });
 
     // Sort the queue in descending order, nothing will mess with the order
-    buildQueue.sort((taskA: ITask, taskB: ITask): number => {
-      return taskB.criticalPathLength! - taskA.criticalPathLength!;
-    });
+    Sort.sortBy(buildQueue, (task: ITask): number => -task.criticalPathLength!);
```

Comment:
(This is probably the riskiest logic change.)

Reply:
The logic looks seems the same here.